documentation/library_guide/introduction.rst

```diff
@@ -70,10 +70,21 @@ Follow the steps below to build your code with |onedpl_short|:
 Below is an example of a command line used to compile code that contains |onedpl_short| parallel algorithms
 on Linux* (depending on the code, parameters within [] could be unnecessary):
 
-.. code:: cpp
+.. code::
 
   icpx [-fsycl] [-fiopenmp] program.cpp [-ltbb] -o program
 
+You may also use the |pstl_offload_option|_ of |dpcpp_cpp| powered by |onedpl_short|
+to build the standard C++ code for execution on a SYCL device:
+
+.. code::
+
+  icpx -fsycl -fsycl-pstl-offload=gpu program.cpp -o program
+
+This option redirects C++ parallel algorithms invoked with the ``std::execution::par_unseq`` policy
+to |onedpl_short| algorithms. It does not change the behavior of the |onedpl_short| execution policies and algorithms
```

Comment:
While I understand (after several reading) that onedpl_short is related to both execution policies and algorithms, perhaps adding it explicitly for the second time helps make this sentence clearer Maybe adding "themselves" also could help. I mean something like:
```suggestion
to |onedpl_short| algorithms. It does not change the behavior of the |onedpl_short| execution policies and |onedpl_short| algorithms themselves
```

Reply:
"... that are directly used in the code" below should convey similar meaning. Anyway, I will think how to make it clearer.

Reply:
Resolved by swapping the words: "It does not change the behavior of the |onedpl_short| algorithms and
execution policies that are directly used in the code."